.github/workflows/node-ci.yml

```diff
@@ -186,9 +186,9 @@ jobs:
           path: |
             .sewing-kit/
             ./packages/*/build/
-          key: ${{ runner.os }}-build-${{ github.sha }}
+          key: ${{ runner.os }}-build-${{ secrets.CACHE_VERSION }}-${{ github.sha }}
           restore-keys: |
-            ${{ runner.os }}-build-
+            ${{ runner.os }}-build-${{ secrets.CACHE_VERSION }}-
```

Comment:
Add a version using GitHub repo environment variable. We can use this to do cache busting going forward